### ISSUE TITLE
When using a screen reader the cell information should read the cell value, column name and index

### ISSUE THREAD

**Issue by simonssspirit** [telerik/kendo-react#564]
When using a screen reader the cell information should read the cell value, column name, and index.

The following official Data Grid example can be used for reference:

https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html

**Comment by Xizario** [comment]
This would be valid in case of single cell selection and/or keyboard navigation. Currently the cells does not have tabIndex by default.

**Comment by kdikov82** [comment]
I have tested the scenario with NVDA and version 5.5.0 and the cell value, column name and index are correctly narrated.
